packages/shared/README.md

```diff
@@ -67,12 +67,47 @@ Use this helper to find an `activeElement` in your component. [Learn more about
 
 Mix `download`, `label`, `href`, and `target` properties into your element to allow it to act more like an `HTMLAnchorElement`.
 
-### ObserverSlotText
+### ObserveSlotPresence
+
+When working with styles that are driven by the conditional presence of `<slot>`s in a component's shadow DOM, you will need to track whether light DOM that is meant for that slot. Use the `ObserveSlotPresence` mixin to target specific light DOM to observe the presence of and trigger `this.requestUpdate()` calls when content fulfilling that selector comes in and out of availability.
```

Comment:
Did you mean:

`you will need to track whether light DOM that is meant for that slot exists`?